llumnix/llm_engine_manager.py

```diff
@@ -250,28 +256,110 @@ async def _migrate(self) -> None:
             logger.error("unexpected exception occurs: {}".format(e))
             logger.error("exception traceback: {}".format(traceback.format_exc()))
 
+    async def rebuild_migrate_backend(self) -> None:
```

Comment:
When we use 'rpc' as migration backend, there is no need to rebuild migration backend, is there any validation in your codes?

Reply:
add it